src/service/poc_lora.proto

```diff
@@ -106,6 +106,12 @@ message lora_valid_beacon_report_v1 {
   // integer representation of a 4-point precision decimal multiplier
   // based on the number of witnesses to a poc event
   uint32 reward_unit = 5;
+  /// the transmit gain value of the gateway in dbi x 10
+  /// For example 1 dbi = 10, 15 dbi = 150
+  /// derived from gateway metadata
+  int32 gain = 6;
+  /// The asserted elevation of the gateway, derived from gateway metadata
+  int32 elevation = 7;
```

Comment:
Declare units relative to what

Reply:
unclear to me what units elevation is in, i imagine its meters but its not defined in any source proto that I am aware off.  Confirm here if you know the units

Reply:
I'm pretty sure that elevation is AGL (above ground level) in meters.